test/ci-operator-configresolver-integration/run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This test runs the ci-operator-configresolver and verifies that it returns
+# properly resolved configs
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOTDIR=$(pwd)
+WORKDIR="$( mktemp -d )"
+trap "rm -rf ${WORKDIR}" EXIT
+
+pushd $WORKDIR
+go build -o tester $ROOTDIR/test/ci-operator-configresolver-integration/main.go
+# copy registry to tmpdir to allow tester to modify registry
+cp -a $ROOTDIR/test/ci-operator-configresolver-integration/ tests
```

Comment:
no need to copy anything.

Reply:
The reason that I'm copying them in is because we will be changing files in the registry/config dirs and verifying that the resolver picks up the changes. It's best to make all those changes from the temp workdir instead of the rootdir.